hwy/base.h

```diff
@@ -903,6 +903,10 @@ using If = typename IfT<Condition, Then, Else>::type;
 // bits explicitly (0x14) instead of attempting to 'negate' 0x102.
 #define HWY_IF_T_SIZE_ONE_OF(T, bit_array) \
   hwy::EnableIf<((size_t{1} << sizeof(T)) & (bit_array)) != 0>* = nullptr
+#define HWY_IF_T_SIZE_LT(T, bytes) \
```

Comment:
Would you mind switching this and its occurrences to _LE, so that it is the negation of GT? We also use LE for V_SIZE and LANES.

Reply:
I have made the change from LT to LE.